packages/block-editor/src/components/editor-skeleton/style.scss

```diff
@@ -77,6 +77,10 @@ html.block-editor-editor-skeleton__html-container {
 .block-editor-editor-skeleton__sidebar {
 	display: none;
 
+	> div {
```

Comment:
I'm not certain this is the right place for this style. Because potentially in other `EditorSkeleton` usage we could have multiple divs showing up one after the other. 

Ideally, this style should be in `packages/edit-post/src/components/sidebar/style.scss` because that div is being created by the withFocusReturn HoC that is added to the Sidebar component. The problem though is there's no way to target this there, it has no className, nothing.

So a middle-ground solution for me is to move this to  `packages/edit-post/src/components/layout/style.scss` because it's edit-post specific.

Reply:
Happy to move it there. Wanted to note that it is almost certainly the addition of this fragment that caused the regression in the first place, and it was then fixed wrong by setting "overflow" to visible on the descendant. We really should be as careful as we can, with adding fragments. Will try to move the CSS now.